tests/cli/test_cli.py

```diff
@@ -488,3 +490,26 @@ def test_run_naive_taskinstance(self, mock_local_job):
             pickle_id=None,
             pool=None,
         )
+
+    @mock.patch("airflow.bin.cli.jobs.SchedulerJob.most_recent_job")
+    def test_health_scheduler(self, mock_most_recent):
+        """
+        Test that the 'scheduler' subcommand for health works
+        """
+        job = SchedulerJob(
+            state='running',
+            latest_heartbeat=timezone.utcnow()
+        )
+        mock_most_recent.return_value = job
+
+        with conf_vars({("scheduler", "scheduler_health_check_threshold"): '60'}):
+            with mock.patch('sys.exit') as mock_exit:
```

Comment:
The cat on this method does not sound good because the question will not be able to use this method either. Can you catch the SystemExit exception instead?